packages/amplify-category-geo/src/__tests__/service-utils/serviceUtils.test.ts

```diff
@@ -109,4 +109,5 @@ describe('Test resource utility functions', () => {
             groupPermissions: groupPermissions
         }).toEqual(geofenceCollectionParams);
     });
+
```

Comment:
@dayjournal could you please revert this? There's no other change in this file and it's better to keep the diff small.

Reply:
The test runs in the local environment and does not seem to be able to erase line breaks.